corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -14,6 +14,7 @@
     UserReportingMetadataStaging,
 )
 from corehq.apps.users.util import (
+    WEIRD_USER_IDS,
```

Comment:
unrelated but that name is weird, `HQ_INTERNAL_USER_IDS` Or `SYSTEM_USER_IDS` would have been better i guess.

Reply:
Yeah i agree with that. I'm not sure on the history of the name, but internal user ids does seem reasonable